.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "third_party/ABY"]
 	path = third_party/ABY
 	url = https://github.com/edwjchen/ABY.git
+[submodule "third_party/SEAL"]
+	path = third_party/SEAL
+	url = git@github.com:microsoft/SEAL.git
```

Comment:
I suspect that the CI user cannot clone using an ssh-based url, since it does not have SSH credentials. Changing this to an https should fix the problem.